packages/experimental/RadioGroup/src/Radio/useRadio.win32.ts

```diff
@@ -0,0 +1,177 @@
+import { RadioProps, RadioInfo } from './Radio.types';
+import * as React from 'react';
+import { useRadioGroupContext } from '../RadioGroup/radioGroupContext';
+import {
+  usePressableState,
+  useOnPressWithFocus,
+  useViewCommandFocus,
+  KeyPressEvent,
+  useKeyDownProps,
+} from '@fluentui-react-native/interactive-hooks';
+import { memoize } from '@fluentui-react-native/framework';
+import { AccessibilityState } from 'react-native';
+
+const defaultAccessibilityActions = [{ name: 'Select' }];
+
+export const useRadio = (props: RadioProps): RadioInfo => {
+  const defaultComponentRef = React.useRef(null);
+
+  // Grabs the context information from RadioGroup (currently selected button and client's onChange callback)
+  const radioGroupContext = useRadioGroupContext();
+
+  const {
+    label,
+    subtext,
+    value,
+    disabled,
+    labelPosition = radioGroupContext.layout === 'horizontal-stacked' ? 'below' : 'after',
+    accessibilityActions,
+    accessibilityLabel,
+    accessibilityHint,
+    accessibilityState,
+    componentRef = defaultComponentRef,
+    accessibilityPositionInSet,
+    accessibilitySetSize,
+    enableFocusRing,
+    ...rest
+  } = props;
+
+  const isDisabled = radioGroupContext.disabled || disabled;
+
+  const buttonRef = useViewCommandFocus(componentRef);
+
+  /* We don't want to call the user's onChange multiple times on the same selection. */
+  const changeSelection = React.useCallback(() => {
+    if (value !== radioGroupContext.value) {
+      radioGroupContext.onChange && radioGroupContext.onChange(value);
+      radioGroupContext.updateSelectedButtonRef && componentRef && radioGroupContext.updateSelectedButtonRef(componentRef);
+    }
+  }, [radioGroupContext, value, componentRef]);
+
+  /* We use the componentRef of the currently selected button to maintain the default tabbable
+    element in a RadioGroup. Since the componentRef isn't generated until after initial render,
+    we must update it once here. */
+  React.useEffect(() => {
+    if (value === radioGroupContext.value && !isDisabled) {
+      radioGroupContext.updateSelectedButtonRef && componentRef && radioGroupContext.updateSelectedButtonRef(componentRef);
+    }
+  }, []);
+
+  // Explicitly only run on mount and unmount
+  React.useEffect(() => {
+    radioGroupContext.addRadioValue(value);
+
+    return () => {
+      radioGroupContext.removeRadioValue(value);
+    };
+  }, []);
+
+  // Explicitly only run on mount and unmount
+  React.useEffect(() => {
```

Comment:
Seems like you could run both `radioGroupContext.addRadioValue(value);` and `radioGroupContext.addRadioEnabledValue(value);` in the same `useEffect`

Reply:
Fixed, thanks!